clients/src/main/java/org/apache/kafka/common/utils/ExponentialBackoff.java

```diff
@@ -47,7 +47,8 @@ public long backoff(long attempts) {
         }
         double exp = Math.min(attempts, this.expMax);
         double term = initialInterval * Math.pow(multiplier, exp);
-        double randomFactor = ThreadLocalRandom.current().nextDouble(1 - jitter, 1 + jitter);
+        double randomFactor = jitter < Double.MIN_NORMAL ? 1.0 :
```

Comment:
I had to look this constant up :) 

Can we just make it check if the jitter is equal to zero (or maybe `<=` zero)? A caller of this method setting jitter to something like 0.5 might be surprised that there is no jitter added.

Reply:
`MIN_NORMAL` is 2^-1022, though.  So it certainly wouldn't affect someone setting jitter = 0.5.

Reply:
Gotcha, I misunderstood the docs and thought it was ~1. Let's keep it as-is